locksmith/src/config/config.ts

```diff
@@ -0,0 +1,54 @@
+import '../utils/envLoader'
+import { Options } from 'sequelize'
+
+const config = {
+  database: {
+    logging: false,
+    dialect: 'postgres',
+  } as Options,
+  stripeSecret: process.env.STRIPE_SECRET,
+  defaultNetwork: process.env.DEFAULT_NETWORK,
+  purchaserCredentials:
+    process.env.PURCHASER_CREDENTIALS ||
+    '0x08491b7e20566b728ce21a07c88b12ed8b785b3826df93a7baceb21ddacf8b61',
+  metadataHost: process.env.METADATA_HOST,
+  unlockApp: process.env.UNLOCK_APP || 'https://app.unlock-protocol.com',
+  logging: false,
+  services: {
+    wedlocks: 'http://localhost:1337',
+  },
+  recaptchaSecret: process.env.RECAPTCHA_SECRET,
+}
+
+if (process.env.ON_HEROKU) {
+  // Heroku needs this:
+  config.database.ssl = true
+  config.database.dialectOptions = {
+    ssl: {
+      require: true,
+      rejectUnauthorized: false,
+    },
+  }
+}
+
+// Database URL
+if (process.env.DATABASE_URL) {
+  const databaseConfigUrl = new URL(process.env.DATABASE_URL)
+  config.database.username = databaseConfigUrl.username
+  config.database.password = databaseConfigUrl.password
+  config.database.host = databaseConfigUrl.hostname
+  config.database.port = Number(databaseConfigUrl.port)
+  config.database.database = databaseConfigUrl.pathname.substring(1)
+} else {
+  config.database.username = process.env.DB_USERNAME
+  config.database.password = process.env.DB_PASSWORD
+  config.database.database = process.env.DB_NAME
+  config.database.host = process.env.DB_HOSTNAME
+}
+
+if (process.env.UNLOCK_ENV === 'prod' || process.env.UNLOCK_ENV === 'staging') {
```

Comment:
Oh I see it here now

Reply:
To explain, I've moved everything to start locksmith to typescript and ESM. 

but we still need the config file as plain cjs for sequelize-cli and using it in migration files which aren't typescript. 

So I duplicated parts I need for tests and migration in config folder. I will think of a way to remove the duplication but it's the most straight forward solution.